packages/frontend-shared/src/utils/spec-utils.ts

```diff
@@ -64,28 +68,65 @@ function collapseEmptyChildren<T extends FoundSpec> (node: SpecTreeNode<T>) {
   return
 }
 
-export function getIndexes (row: UseCollapsibleTreeNode<SpecTreeNode<FuzzyFoundSpec>>) {
-  const indexes = row.data?.indexes || []
+export function getDirIndexes (row: UseCollapsibleTreeNode<SpecTreeNode<FuzzyFoundSpec>>) {
+  const indexes = row.data?.dirIndexes ?? []
 
   const maxIndex = row.id.length - 1
   const minIndex = maxIndex - row.name.length + 1
 
-  const res = indexes?.filter((index) => index >= minIndex && index <= maxIndex)
+  const res = indexes.filter((index) => index >= minIndex && index <= maxIndex)
 
   return res.map((idx) => idx - minIndex)
 }
 
+export function fuzzySortSpecs (specs: FuzzyFoundSpec[], searchValue: string) {
+  const transformedSpecs = addDirectoryToSpecs(specs)
+
+  return fuzzySort
+  .go(searchValue, transformedSpecs, { keys: ['baseName', 'directory'], allowTypo: false })
```

Comment:
Instead of transforming the spec and adding the directory, can't we just add `relative` to the `keys` array?

Reply:
That's what we were doing at first but if it was one string then if you searched something like runner it would show the runner directory and not the files with runner in them as well because it already found runner earlier in the string.